eth2util/eth2exp/attagg.go

```diff
@@ -149,3 +158,79 @@ type BeaconCommitteeSubscriptionResponse struct {
 	// IsAggregator indicates whether the validator is an attestation aggregator.
 	IsAggregator bool
 }
+
+// CalculateCommitteeSubscriptionResponse returns a BeaconCommitteeSubscriptionResponse with IsAggregator field set to true if the validator is an aggregator.
+func CalculateCommitteeSubscriptionResponse(ctx context.Context, eth2Svc eth2client.Service, subscription BeaconCommitteeSubscription) (BeaconCommitteeSubscriptionResponse, error) {
```

Comment:
I think we decided on using pointers for these types as that is the eth2client pattern and we plan to migrate to it eventually.